### DISCUSSION TITLE
No switch in extension setting page

### DISCUSSION THREAD

**Discussion by wea-f** [3kh0/ext-remover#762] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
The extension that I want to disable doesn't have any switch in the setting page, does this mean I can't do it or I just skip the step of flipping the switch?

![Screenshot 2023-10-31 9 50 50 PM](https://github.com/3kh0/ext-remover/assets/100387337/5e713aa5-9f96-4520-8609-c830ae318d00)

**Comment by S-PScripts** [comment]
you cant do it
